tools/download-test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+while [[ $# -gt 1 ]]
+do
+  key="$1"
+
+  case "$key" in
+      -t|--tap)
+      TAP="$2"
+      shift # past argument
+      ;;
+      *) # unknown option
+      ;;
+  esac
+  shift # past argument or value
+done
+
+if [ "$DOWNLOAD_LOCATION" == "Nightly" ]; then
+	DOWNLOAD_DIR="nightly"
+else
+	DOWNLOAD_DIR="release"
+fi
+
+LINK=`rsync rsync://unencrypted.nodejs.org/nodejs/$DOWNLOAD_DIR/ | grep $NODE_VERSION | sort -t. -k 1,1n -k 2,2n -k 3,3n | tail -1 | awk '{print $5}'`
```

Comment:
`LINK` is really `nodeVersion` right? Could you add a comment along the lines of:

```bash
# If the provided NODE_VERSION is inexact, e.g. `v8`, get the most recent matching version.
```